### PR TITLE
[JA] Translations (Privacy Sandbox, Blog)

### PR DESCRIPTION
**Pull request by ilyaspiridonov**
Japanese translations for: 
[blog/privacy-sandbox-unified-origin-trial/index.md](https://gitlocalize.com/repo/7832/ja/review/49452)
[docs/privacy-sandbox/attribution-reporting/system-overview/index.md](https://gitlocalize.com/repo/7832/ja/review/49451)
[blog/user-agent-reduction-deprecation-trial/index.md](https://gitlocalize.com/repo/7832/ja/review/49449)
[blog/chips-origin-trial-extended/index.md](https://gitlocalize.com/repo/7832/ja/review/49448)
[blog/chips-origin-trial/index.md](https://gitlocalize.com/repo/7832/ja/review/49447)
UPD: [/blog/user-agent-reduction-origin-trial/index.md](https://gitlocalize.com/repo/7832/ja/review/49450)
Translated by Alconost (MTPE)

### CODE REVIEW

**Comment by ilyaspiridonov** [comment]
CC: @agektmr 
Please note there's one last [article pending your review](https://gitlocalize.com/repo/7832/ja/review)

**Comment by agektmr** [comment]
I left a comment in the remaining one article. Could you please check that? It might be a bug.

File: site/ja/docs/privacy-sandbox/attribution-reporting/system-overview/index.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+## ウェブサイトからブラウザへの通信 {: #web-browser-communication }
+
+{% Aside %} この API はテスト中ではありますが、コードで API が使用可能であることを確認し、適切な Permissions-Policy を設定する必要があります。 {% endAside %}
+
+<figure class="screenshot">{% Img src="image/VbsHyyQopiec0718rMq2kTE1hke2/wO7uPRjjBsSoUGNN8M9B.png", alt="サイト運営者のウェブサイトのアトリビューション ソースは、広告主のウェブサイトのトリガーに接続します。", width="800", height="275" %}<figcaption><strong>図 1</strong> .</figcaption></figure>
+
+### アトリビューション ソース（サイト運営者のウェブサイト） {: #attribution-sources}
+
+*[アトリビューション ソース](https://docs.google.com/document/d/1BXchEk-UMgcr2fpjfXrQ3D8VhTR-COGYS1cwK_nyLfg/edit#heading=h.dmy7iuqnhvby)*は、広告関連のイベント（クリックまたはビュー）であり、アドテックは以下の種類の情報を添付できます。
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/attribution-reporting/system-overview/index.md:71]:
```suggestion
*[アトリビューション ソース](https://docs.google.com/document/d/1BXchEk-UMgcr2fpjfXrQ3D8VhTR-COGYS1cwK_nyLfg/edit#heading=h.dmy7iuqnhvby)* は、広告関連のイベント（クリックまたはビュー）であり、アドテックは以下の種類の情報を添付できます。
```

File: site/ja/docs/privacy-sandbox/attribution-reporting/system-overview/index.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+*フィルター*（またはカスタムルール）を設定して、トリガーが特定のソースに一致するタイミングを決定できます。たとえば、特定の商品カテゴリのコンバージョンのみをカウントし、他のすべてのカテゴリを無視するようにフィルターを設定します。フィルターと優先順位付けモデルにより、より高度なアトリビューション レポートが可能になります。
+
+ローカル ストレージに複数の属性ソースが見つかった場合、ブラウザは最近保存されたものを選択します。アトリビューション ソースに優先度が割り当てられている場合、ブラウザは最も優先度の高いソースを選択します。
+
+## データ収集
+
+同時に、対応するソースに一致するアトリビューション トリガーは、ブラウザによってアドテックが所有するサーバー上のレポートエンドポイント（収集エンドポイントまたは収集サービスと呼ばれることもあります）にレポートとして送信されます。これらのレポートは、イベントレベルレポートまたは集計可能なレポートです。
+
+*[集計可能なレポート](https://github.com/WICG/conversion-measurement-api/blob/main/AGGREGATE.md#aggregatable-reports)*は、要約レポートの生成に使用されます。集計可能なレポートは、（サイト運営者のサイトで）広告から収集されたデータと（広告主のサイトからの）コンバージョンデータを組み合わせたもので、アドテックによって収集される前に、ユーザーのデバイスのブラウザによって生成され暗号化されます。
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/attribution-reporting/system-overview/index.md:100]:
```suggestion
*[集計可能なレポート](https://github.com/WICG/conversion-measurement-api/blob/main/AGGREGATE.md#aggregatable-reports)* は、要約レポートの生成に使用されます。集計可能なレポートは、（サイト運営者のサイトで）広告から収集されたデータと（広告主のサイトからの）コンバージョンデータを組み合わせたもので、アドテックによって収集される前に、ユーザーのデバイスのブラウザによって生成され暗号化されます。
```

File: site/ja/docs/privacy-sandbox/attribution-reporting/system-overview/index.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+
+イベントレベルレポートには 2 ～ 30 日の遅延があります。集計可能なレポートは 1 時間以内にランダムな遅延で送信され、イベントは*[コントリビューション予算](https://github.com/WICG/attribution-reporting-api/blob/main/AGGREGATE.md#contribution-bounding-and-budgeting)*内に収まる必要があります。これらの選択により、プライバシーが保護され、個々のユーザーのアクションの悪用が防止されます。
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/attribution-reporting/system-overview/index.md:102]:
```suggestion
イベントレベルレポートには 2 ～ 30 日の遅延があります。集計可能なレポートは 1 時間以内にランダムな遅延で送信され、イベントは *[コントリビューション予算](https://github.com/WICG/attribution-reporting-api/blob/main/AGGREGATE.md#contribution-bounding-and-budgeting)* 内に収まる必要があります。これらの選択により、プライバシーが保護され、個々のユーザーのアクションの悪用が防止されます。
```

File: site/ja/docs/privacy-sandbox/attribution-reporting/system-overview/index.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+*[アトリビューション ソース](https://docs.google.com/document/d/1BXchEk-UMgcr2fpjfXrQ3D8VhTR-COGYS1cwK_nyLfg/edit#heading=h.dmy7iuqnhvby)*は、広告関連のイベント（クリックまたはビュー）であり、アドテックは以下の種類の情報を添付できます。
+
+- 広告クリエイティブ ID、キャンペーンに関する情報、地理情報などのコンテキストレポートデータ。
+- ユーザーに終着してほしい広告のターゲットであるサイトのコンバージョンターゲット。これにより、特定のコンバージョンに対応するソースイベントがブラウザに通知されます。
+
+アトリビューション ソースのデータが収集されると、ブラウザはデータをローカルストレージに追加するため、後でアトリビューション トリガーと照合できます。
+
+### アトリビューション トリガー（広告主のウェブサイト）
+
+*[アトリビューション トリガー](https://github.com/WICG/attribution-reporting-api/blob/main/AGGREGATE.md#attribution-trigger-registration)*は、コンバージョンを取得するよう API に指示するイベントです。
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/attribution-reporting/system-overview/index.md:80]:
```suggestion
*[アトリビューション トリガー](https://github.com/WICG/attribution-reporting-api/blob/main/AGGREGATE.md#attribution-trigger-registration)* は、コンバージョンを取得するよう API に指示するイベントです。
```

**Comment by ilyaspiridonov** [comment]
> I left a comment in the remaining one article. Could you please check that? It might be a bug.

Yes, I did see it and fixed it. It's not a bug, the translator just removed one opening { by mistake

**Comment by agektmr** [comment]
I just saw your comment at GitLocalize. Is it possible to receive an email when there's a reply?

File: site/ja/blog/privacy-sandbox-unified-origin-trial/index.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+- 標準のブラウジング セッションを開始しており、**シークレット モードは使用していない**こと。
+- **Chrome のアクティブな実験グループ**に属していること。
+
+## 開発者の機能が API サポートを検出する仕組み
+
+他のウェブプラットフォーム機能と同様に、使用する前に機能がサポートされていることを検出する必要があります。
+
+### Topics
+
+`document` 内の `browsingTopics()` 関数と、「browsing-topics」のアクセス許可ポリシー（/docs/privacy-sandbox/permissions-policy/#featurepolicyallowsfeaturefeature）を確認してください。
```
Comment (agektmr) [inline site/ja/blog/privacy-sandbox-unified-origin-trial/index.md:104]: This is a bug in the original article but I'd rather fix it in Japanese.

```suggestion
`document` 内の `browsingTopics()` 関数と、「browsing-topics」の[アクセス許可ポリシー](/docs/privacy-sandbox/permissions-policy/#featurepolicyallowsfeaturefeature)を確認してください。
```

**Review by agektmr — CHANGES_REQUESTED** [review]

**Comment by ilyaspiridonov** [comment]
> I just saw your comment at GitLocalize. Is it possible to receive an email when there's a reply?
We do have this on our roadmap )

**Comment by ilyaspiridonov** [comment]
@agektmr I have added [the latest file](https://gitlocalize.com/repo/7832/ja/review/49450) to this PR as well

**Review by agektmr — APPROVED** [review]
LGTM

**Pull request merged**